packages/blade/src/components/Tooltip/Tooltip.stories.tsx

```diff
@@ -0,0 +1,290 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
+/* eslint-disable @typescript-eslint/explicit-function-return-type */
+import type { ComponentStory, Meta } from '@storybook/react';
+import React from 'react';
+import { Title } from '@storybook/addon-docs';
+import type { TooltipTriggerProps } from './types';
+import type { TooltipProps } from './';
+import { TooltipInteractiveWrapper, Tooltip as TooltipComponent } from './';
+import { Button } from '~components/Button';
+import { BankIcon, InfoIcon } from '~components/Icons';
+import { Link } from '~components/Link';
+import { Box } from '~components/Box';
+import { Text } from '~components/Typography';
+import { isReactNative } from '~utils';
+import { List, ListItem } from '~components/List';
+import { IconButton } from '~components/Button/IconButton';
+import StoryPageWrapper from '~src/_helpers/storybook/StoryPageWrapper';
+import { Sandbox } from '~src/_helpers/storybook/Sandbox';
+
```

Comment:
we shall also add a notice here to install the peerDependencies for making this work. also we shall add that to bottomsheet.